src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
Most of these changes landed in 76df9e4e389d80bff410a9e5f08b848de1d21a2f. AFAIK @gibson042 advocated creating variables for every one of them so maybe he'll want to say sth here.